bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/ModifiablePersistenceService.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.persistence;
+
+import java.util.Date;
+
+import org.eclipse.smarthome.core.items.Item;
+
+/**
+ * This class provides an interface to the a {@link PersistenceService} to allow data to be stored
+ * at a specific time. This allows bindings that interface to devices that store data internally,
+ * and then periodically provide it to the server to be accommodated.
+ *
+ * @author Chris Jackson - Initial implementation and API
+ *
+ */
+public interface ModifiablePersistenceService extends QueryablePersistenceService {
+    /**
+     * <p>
+     * Stores the historic item value. This allows the item, time and value to be specified.
+     * </p>
+     * <p>
+     * Implementors should keep in mind that all registered {@link PersistenceService}s are called synchronously. Hence
+     * long running operations should be processed asynchronously. E.g. <code>store</code> adds things to a queue which
+     * is processed by some asynchronous workers (Quartz Job, Thread, etc.).
+     * </p>
+     *
+     * @param date the date of the record
+     * @param item the data to be stored
+     */
+    public void store(Date date, Item item);
```

Comment:
If you don't care, I would prefer to use Item as first argument and Date as second one.
Why? Because there is already the method `void store(Item item);`.
https://github.com/eclipse/smarthome/blob/e0985e9/bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/PersistenceService.java#L40
And if this one has an additional argument I prefer to add it at the endt.